tests/e2e/autoscaling.cpu-sys-fs-state-scaling/00-create-vm.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       min: 1
       max: 5
       use: 1
-    memoryProvider: DIMMSlots
+    memoryProvider: VirtioMem
```

Comment:
Should this change be merged into the PR this is based on?

(+ same question for the other changes to this test)

Reply:
Fair enough, makes sense